backend/src/db/factory/user.ts

```diff
@@ -0,0 +1,13 @@
+import { faker } from '@faker-js/faker/locale/en_US';
```

Comment:
would prefer to export this from a faker.ts file so people don't use auto-import and get the whole module in another file